telosnetwork/telos-wallet
# Some nfts are not shown because they lack an attribute

# Fixes #668 

## Description
The Pull Request addresses the issue where some NFTs were not being displayed in the Telos Wallet inventory. This was due to these NFTs lacking the `supportedInterfaces: string[]` attribute, which is required for filtering and displaying them. The PR likely involves modifications to handle NFTs that might not have this attribute, ensuring they are still visible in the inventory.

## Test Scenarios
- Go to [this Testnet link](https://deploy-preview-669--wallet-staging.netlify.app/) and login using the Teams Account
- Check the inventory for the NFTs from the screenshot (lacking the `supportedInterfaces` attribute)
- Verify all NFTs, including those without the `supportedInterfaces` attribute, are now displayed

 ## Screenshot

This is the screenshot of three Items created with no supportedInterfaces attribute. You should see them in the Team Account inventory.
 
![image](https://github.com/telosnetwork/telos-wallet/assets/4420760/eb60b673-f927-4d74-b0a2-b44547f33882)


## Checklist:
-   [x] I have included all English text in the translation file and/or created a new issue with the required translations for the currently supported languages
